index.html

```diff
@@ -253,7 +157,67 @@ <h2 class="contact--me">Contact me</h2>
     </form>
   </section>
   <section class="pop-up" id="modal">
-    
```

Comment:
- [ ] 🙏Please update the code to dynamically populate the data for the pop-up display. Currently, it only displays the same data every time you click. This modification will allow users👨‍👩‍👧‍👦 to view the details of each specific project when they click the "See Project" button.